config/esw.yaml

```diff
@@ -111,6 +111,40 @@ brushed_motors:
       inversion: 1.0
   i2c_device_file: "/dev/i2c-1"
   test: false
+  
+cameras:
```

Comment:
New params: 1) 1280x720, 30fps, 4000000bps 2) 640x480, 15fps, 1500000bps 3) 320x240, 15fps, 460000bps

Reply:
These numbers don't seem to line up with what we had before, like 1280x720 at 30fps had a different bitrate. Just want to make sure this is definitely accurate?

Reply:
Yeah, they've gone up slightly due to 2.4GHz giving us higher bandwidth. Right now they're 4Mbps 1.5Mbps and 460kbps. The last two should work fine but if we find we're throttling the connection with 4Mbps we can drop it to 3.2Mbps (the old value I think). For 900MHz then I'd advise against two concurrent max-res streams.

Reply:
The settings that were being run for the mission today also seemed pretty good. We were running 4.2Mbps max res for cameras at certain points.

Reply:
Okay so if what we had today was fine should we merge? Or switch to what you said initially in this comment thread?

Reply:
I offered 3 params mainly bc I didn't think we wanted to merge with 5. I imagine only having 3 to choose from would simplify it for the operators.

Reply:
Personally I don't mind having more options, I just want to make sure the nitrate lines up with the other parameters as you expect, cause I don't really understand how those are calculated

Reply:
sg to me. Bitrates aren't really calculated using any rigorous or sophisticated method. The highest bitrate setting (~4Mbps) is roughly in line with the bitrate you'd expect from an uncompressed or minimally compressed 1280x720 30fps footage (source: quick Google search). The lower bitrate settings are there for when the feed degrades over distance (e.g. dropped packets, frame freezing, excessive compression artifacts, etc.).